src/main/java/fileio_use_case/interface_adapters/SessionFileController.java

```diff
@@ -13,22 +12,21 @@
  * for the program.
  */
 
-public class SessionFileController {
+public class SessionFileController{
     private final SessionFileImportInputBoundary interactor;
     public SessionFileController(SessionFileImportInputBoundary interactor) {
         this.interactor = interactor;
     }
 
     /**
      * Takes in file path of Session, creates request model, and pass request model to the interactor
-     * to create fall and winter session files.
-     * REPRESENTATION INVARIANT: file contains both fall and winter courses
+     * to create fall or winter sessions.
+     * REPRESENTATION INVARIANT: file contains fall and/or winter courses
      * @param filePath - File path to JSON file for session
-     * @return SessionModel
      */
-    public SessionModel createSessionFile(String filePath, String sessionType) throws IOException, ParseException, java.text.ParseException, InvalidSectionsException {
+    public void createSessionFile(String filePath, String sessionType) throws IOException, ParseException, java.text.ParseException, InvalidSectionsException {
         FileImportRequestModel requestModel = new FileImportRequestModel(filePath);
```

Comment:
Return values might be a good idea purely to facilitate or ease testing, although there's good reason to remove it. Small comment to consider.

Reply:
emily